.stylelintrc.js

```diff
@@ -14,5 +14,8 @@ module.exports = {
     'carbon/motion-easing-use': true,
     'carbon/theme-use': true,
     'carbon/type-use': true,
+
+    // CSS Logical properties
+    'csstools/use-logical': 'ignore',
```

Comment:
Again, ignoring this for now, until we successfully change all physical properties to logical, then we will add a follow-up PR to enable this. 

The reason is so that new created PR's branching off of main do not have hundreds of stylelint errors